### ISSUE TITLE
Abstract and Introduction - info elsewhere

### ISSUE THREAD

**Issue by slhenry** [w3c/silver#157]
Much of the information in the Introduction should be provided elsewhere -- probably in a page on the WAI website -- perhaps some in [WCAG Overview](https://www.w3.org/WAI/standards-guidelines/wcag/), some in [WCAG FAQ](https://www.w3.org/WAI/standards-guidelines/wcag/faq/), &/or a new page on WCAG 3.0. This /TR/ document can then include link(s) to point to that information elsewhere.

The Abstract is too long. Also, it should be a summary -- that is, information should not be *only* in the Abstract, which some currently is.

I think OK to leave as is for FPWD, and hope that we can work together to improve it in next draft. :-)
